packages/main/src/Link.js

```diff
@@ -241,6 +254,9 @@ class Link extends UI5Element {
 	}
 
 	get tabIndex() {
+		if (this._tabIndex.length) {
```

Comment:
it will be undefined, -1 (valid and will be set) or 0 (valid and will be set)  so we don't need to check length here

Reply:
Done